manifests/i/IDRIX/DirHash/1.24.0/IDRIX.DirHash.locale.en-US.yaml

```diff
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: IDRIX.DirHash
+PackageVersion: 1.24.0
+PackageLocale: en-US
+Publisher: IDRIX
+PublisherUrl: https://www.idrix.fr/
+PublisherSupportUrl: https://github.com/idrassi/DirHash/issues/
+# PrivacyUrl:
+Author: Mounir IDRASSI
+PackageName: DirHash
+PackageUrl: https://idrassi.github.io/DirHash/
```

Comment:
```suggestion
PackageUrl: https://www.idrix.fr/Root/content/category/7/31/55/
```

Reply:
This helps justify IDRIX as the ISV, since the GitHub page appears to be the personal page of Mounir IDRASSI when viewed in isolation.